src/settings/toml/tests/mod.rs

```diff
@@ -76,6 +85,66 @@ fn it_builds_from_environments_config_with_kv() {
     }
 }
 
+#[test]
+fn parses_same_from_config_path_as_string() {
```

Comment:
here's the test for asserting that manifest from string works comparably to manifest from config path.